gap/examples.gi

```diff
@@ -371,6 +371,55 @@ GeneralisedPetersenGraphCons);
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
 
+InstallMethod(KingsGraphCons,
+"for IsMutableDigraph and two positive integers",
+[IsMutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D, i, j;
+  D := EmptyDigraph(filt, n * k);
+  for i in [1 .. k] do
+    for j in [1 .. (n - 1)] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + 1]);
+    od;
+  od;
+  for i in [1 .. (k - 1)] do
+    for j in [1 .. n] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + n]);
+    od;
+  od;
+  for i in [1 .. (k - 1)] do
+    for j in [1 .. (n - 1)] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + (n + 1)]);
+      DigraphAddEdge(D, [((i - 1) * n) + j + 1, ((i - 1) * n) + j + n]);
+    od;
+  od;
+  return DigraphSymmetricClosure(D);
+end);
+
+InstallMethod(KingsGraphCons,
+"for IsImmutableDigraph and two positive integers",
+[IsImmutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D;
+  D := MakeImmutable(KingsGraphCons(IsMutableDigraph, n, k));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  SetIsConnectedDigraph(D, true);
+  SetIsBipartiteDigraph(D, n * k in Difference([n, k], [1]));
+  SetIsPlanarDigraph(D, true);
```

Comment:
Oh actually, sorry, I'm wrong about planarity! It'll be planar if n=1 or k=1 or n=k=2... what about any other combinations? If you can't work it out in a nice way, then feel free to delete this line.

Reply:
Perhaps it's planar if and only if `n <= 2 or k <= 2 or n = k = 3`? That's my current best guess. If you can prove this, (or prove it to be something else), then please put in the correct condition. But don't spend too long thinking about it - we can easily add it later if we subsequently work it out.

Reply:
Right, yeah. I think the rule is probably just that it's planar when n <= 2 or k <= 2, so I'll add that condition.

Reply:
...and on further consideration n=k=3 is also planar so I'll add that too

Reply:
`n = k = 3` is also definitely planar, but I think that's the only other exception.